public/stylesheets/styles.css

```diff
@@ -5,3 +5,119 @@ dark-blue: #0a2c42;
 grey: #bfbfbf;
 font-family: Arial, Helvetica, sans-serif;
 */
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+header,
+footer {
+  flex: 0 0 content;
```

Comment:
This causes a warning "Invalid property value" if you check it in the developer tools.